tests/test_models/test_huggingface.py

```diff
@@ -153,5 +154,3 @@ def test_one_text_two_targets(self):
 
     # TODO: add test with long text input, e.g. thousands of words,
     #  to see if we need batching, and to stress-test token alignment
```

Comment:
Other edge cases I'd want to test

- with the sorts of "word" input representations we see in RT corpora. e.g. punctuation and word still attached

```python
['If',
 'you',
 'were',
 'to',
 'journey',
 'to',
 'the',
 'North',
 'of',
 'England,',
 'you',
 'would',
 'come',
 'to',
 'a',
 'valley',
 'that',
 'is',
 'surrounded',
 'by',
 'moors',
 'as',
 'high',
 'as',
 'mountains.']
```

(I have to re-check how we most recently decided to handle surprisal of punctuation tokens in the aggregation; I'll get back to you on this.)

- use a model that attaches BOS/EOS tokens, e.g. `xlm-roberta`

```python
In [12]: t = transformers.AutoTokenizer.from_pretrained("xlm-roberta-base")

In [14]: t.convert_ids_to_tokens(t("this is a test").input_ids)
Out[14]: ['<s>', '▁this', '▁is', '▁a', '▁test', '</s>']
```